charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -476,6 +476,8 @@ volumeSnapshotClasses: []
 # Intended for use with older clusters that cannot easily replace the CSIDriver object
 # This parameter should always be false for new installations
 useOldCSIDriver: false
+# Deploy EBS CSI Driver without controller and associated resources
+disableController: false
```

Comment:
Anyone got a better name or location for this? I believe we need to have named such that it defaults to false due to internal reasons.

My other thought was top level `NodeDaemonsetOnly` OR a `controller.disabled`

Either way I agree that having it default to false makes all the if not's in each template ugly :(

Reply:
I like controller.disabled or controllerIsDisabled